server/models/UserModel/index.js

```diff
@@ -11,6 +11,8 @@ const userMongoProjection = {
 }
 
 const UserModel = {
+  all: async (db) =>
+    db.collection(collections.users).find({}, userMongoProjection).toArray(),
```

Comment:
Does every admin user have access to all users, including other admins, and should they include themselves in the result..?